From 9f5975026d3c80e1bda2b2adcbe17fd4e60c7d21 Mon Sep 17 00:00:00 2001
From: Geoff Thorpe <Geoff.Thorpe@freescale.com>
Date: Sat, 30 Mar 2013 04:17:59 -0500
Subject: [PATCH 247/547] powerpc/85xx: Add platform support for the USDPAA
 driver

Signed-off-by: Geoff Thorpe <Geoff.Thorpe@freescale.com>
Signed-off-by: Emil Medve <Emilian.Medve@Freescale.com>
[Original patch taken from QorIQ-SDK-V1.4-SOURCE-20130625-yocto.iso]
Signed-off-by: Bin Jiang <bin.jiang@windriver.com>
---
 arch/powerpc/platforms/85xx/corenet_ds.c | 5 +++++
 1 file changed, 5 insertions(+)

diff --git a/arch/powerpc/platforms/85xx/corenet_ds.c b/arch/powerpc/platforms/85xx/corenet_ds.c
index c4333c5..557ad0b 100644
--- a/arch/powerpc/platforms/85xx/corenet_ds.c
+++ b/arch/powerpc/platforms/85xx/corenet_ds.c
@@ -31,6 +31,8 @@
 #include <sysdev/fsl_pci.h>
 #include "smp.h"
 
+#include <linux/fsl_usdpaa.h>
+
 void __init corenet_ds_pic_init(void)
 {
 	struct mpic *mpic;
@@ -131,4 +133,7 @@ __init void corenet_ds_init_early(void)
 #ifdef CONFIG_FSL_BMAN_CONFIG
 	bman_init_early();
 #endif
+#ifdef CONFIG_FSL_USDPAA
+	fsl_usdpaa_init_early();
+#endif
 }
-- 
1.8.4.93.g57e4c17

